app/src/main/java/agency/five/codebase/android/movieapp/ui/component/FavoriteButton.kt

```diff
@@ -0,0 +1,42 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.ui.theme.Blue
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun FavoriteButton(modifier: Modifier = Modifier) {
+    val isFavorite = remember { mutableStateOf(true) }
+
+    Image(
+        painter = painterResource(
+            id = if(isFavorite.value)
+                R.drawable.heart_full
+            else
+                R.drawable.heart_empty),
+        contentDescription = "Favorite",
+        modifier = Modifier
```

Comment:
Here you are also not using provided modifier, instead of creating new one with `Modifier`, just use the provided one and build on top of it, i.e. leave everything else as is, just replace `Modifier` with `modifier`.